source/_integrations/iron_os.markdown

```diff
@@ -40,6 +40,13 @@ Home Assistant will detect nearby IronOS devices. Discovered devices will show u
 
 {% include integrations/config_flow.md %}
 
+## Configuration parameters
+
+{% configuration_basic %}
+"Address":
+  description: "The Bluetooth address of the device. This is automatically detected by Home Assistant, so manual entry is not necessary."
```

Comment:
> manual entry is not necessary

Does the config flow allow manual entry? Glancing at the code, it appears to be a list: `vol.In(self._discovered_devices)` which would not allow the user to type anything in--only select an option.

Would a description like this be better/more clear?
> The physical address of the detected Bluetooth device.

Reply:
physical as oposed to virtual address? Not sure if that is the right term. The official term is Bluetooth device address (BD_ADDR).

Reply:
I wanted to point out that this isn't a value the user has to type in, in fact, in the config flow the user won't ever see the address, the label of the select displays only the device name. How do other ble integrations describe the address field in the documentation?